### ISSUE TITLE
Upgrade to 3.14.0 fails due to 404 not found

### ISSUE THREAD

**Issue by MayeulC** [YunoHost-Apps/monitorix_ynh#45]
http://www.monitorix.org/monitorix_3.14.0-izzy1_all.deb returns 404

https://paste.yunohost.org/raw/rizuwakade

```
  YNH_APP_CURRENT_VERSION: 3.12.0~ynh2
  YNH_APP_MANIFEST_VERSION: 3.14.0~ynh2
  YNH_APP_UPGRADE_TYPE: UPGRADE_APP
  YNH_ARCH: amd64
error: An error occurred inside the app upgrade script
```

**Comment by olberger** [comment]
Indeed, the package is now at https://www.monitorix.org/old-versions/monitorix_3.14.0-izzy1_all.deb

**Comment by Josue-T** [comment]
Closed by #44
